netsync/manager.go

```diff
@@ -1095,7 +1095,7 @@ func (sm *SyncManager) handleInvMsg(imsg *invMsg) {
 
 	// Ignore invs from peers that aren't the sync if we are not current.
 	// Helps prevent fetching a mass of orphans.
-	if peer != sm.syncPeer && !sm.current() {
+	if !sm.current() && peer != sm.syncPeer {
```

Comment:
Can you please provide some more context explaining why this change is necessary?